tracer/src/Datadog.Trace/Generated/net6.0/Datadog.Trace.SourceGenerators/AspectsDefinitionsGenerator/AspectsDefinitions.g.cs

```diff
@@ -550,5 +550,343 @@ internal static partial class AspectDefinitions
 "  [AspectMethodReplace(\"System.Text.StringBuilder::AppendJoin(System.Char,System.Collections.Generic.IEnumerable`1<!!0>)\",\"\",[0],[False],[None],Default,[])] AppendJoin(System.Text.StringBuilder,System.Char,System.Object)",
 "  [AspectMethodReplace(\"System.Text.StringBuilder::AppendJoin(System.String,System.Collections.Generic.IEnumerable`1<!!0>)\",\"\",[0],[False],[None],Default,[])] AppendJoin(System.Text.StringBuilder,System.String,System.Object)",
         };
+
+        public static string[] RaspAspects = new string[] {
```

Comment:
Just realised, both this and the IAST aspects are a static field. Doesn't that mean they'll _both_ be allocated, whichever we need? 🤔 They're only ever invoked once right? If so, maybe we should change them both to methods that return the array instead? e.g.:

```suggestion
        public static string[] GetRaspAspects() => new string[] {
```

Reply:
Yes, that's a very good idea. I have updated the code. Thanks!